src/libslic3r/PrintObject.cpp

```diff
@@ -152,6 +152,146 @@ std::vector<std::reference_wrapper<const PrintRegion>> PrintObject::all_regions(
     return out;
 }
 
+//SS:Polyholes
```

Comment:
Can we change the comment to something like: Convert holes to Polyholes, implementation is ported from SuperSlicer? 
It will be easier to understand.

And remove `SS:Polyholes` comments from other places to make the comments style consistent?

Reply:
Just confirm you want the bulk of the SS:Polyholes comment/section tags removed, happy to add key comments around the entry point and main code.

Reply:
Yes.
For key implementations, like this one, it's preferable to add more descriptive comments for clarity.

In other areas, we can use the `// Orca` comment, which assists me during extensive merge tasks. 
It's also fine to skip adding a comment in those place if you prefer